<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMode</class>
 <widget class="QWidget" name="UserMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserMode</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>991</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="header">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Watch List:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="watchList"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonsLayout">
        <item>
         <widget class="QLabel" name="titleLabel">
          <property name="text">
           <string>Title:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="titleLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="genreLabel">
          <property name="text">
           <string>Genre:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="genreLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="inputLayout">
        <item>
         <widget class="QPushButton" name="removeButton">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filterButton">
          <property name="text">
           <string>Filter by genre</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="desiredGenre">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Desired genre:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="genreList"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="currentLabel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Current movie:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="currentMovie"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextButton">
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
